### DISCUSSION TITLE
Do I need the Redirect URI when I'm signing up for the API?

### DISCUSSION THREAD

**Discussion by JB184351** [trakt/trakt-api#433] [Q&A] [answered]
Hi, I want to work on an iOS app using this API but it's been awhile since working with APIs, I'm wondering what the redirect URI is for and do I need my own website to use this API? Mostly want to make an app for learning purposes.

<img width="950" alt="Screenshot 2024-01-22 at 5 52 05 PM" src="https://github.com/trakt/api-help/assets/36495451/2bb49800-1391-40e3-8e1e-7579527076f5">

**Comment by rectifyer** [comment, marked as answer]
You can use the url on the 2nd line if you don't have your own server.

**Reply by JB184351** [reply]
Oh awesome thanks!
